kinova_tracking/pos_kinova_arm.py

```diff
@@ -154,24 +44,37 @@ def prepare_ocp(
         "thoraxRT5": my_ik.q[1, :].mean(),
         "thoraxRT6": my_ik.q[2, :].mean(),
     }
-    old_biomod_file = (
-        "../models/KINOVA_merge_without_floating_base_template.bioMod"
-    )
-    new_biomod_file = (
-        "../models/KINOVA_merge_without_floating_base_template_with_variables.bioMod"
-    )
+    old_biomod_file = "../models/KINOVA_merge_without_floating_base_template.bioMod"
+    new_biomod_file = "../models/KINOVA_merge_without_floating_base_template_with_variables.bioMod"
     add_header(old_biomod_file, new_biomod_file, thorax_values)
 
     model_path = new_biomod_file
 
     biorbd_model = biorbd.Model(model_path)
 
-    q0_rescue = np.array([
-                -0.18905905, -0.22494988,  0.17328238,  0.22634287,  0.0068234, 0.18016999,  0.68947024,
-                -0.01252137,  1.10091368,  1.0030519, 0.,  0.2618, 0.3903, 1.7951, 0.6878, 0.3952])
+    q0_rescue = np.array(
+        [
+            -0.18905905,
+            -0.22494988,
+            0.17328238,
+            0.22634287,
+            0.0068234,
+            0.18016999,
+            0.68947024,
+            -0.01252137,
+            1.10091368,
+            1.0030519,
+            0.0,
+            0.2618,
+            0.3903,
+            1.7951,
+            0.6878,
+            0.3952,
+        ]
+    )
 
     q0_1 = my_ik.q[6:, 0]
-    q0_2 = np.array((-0.2892, 0.6695, 0.721, 0.0, 0.0, 0.0))
+    # q0_2 = np.array((-0.2892, 0.6695, 0.721, 0.0, 0.0, 0.0))
```

Comment:
still necessary ?

Reply:
for this case yes, we didn't make an inverse kinematics with a merged biomod